### PR TITLE
Uyuni server upgrade keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

We also need a dedicated intro text for the Uyuni Server upgrade

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
> What about the other change regarding the migration script?

It's gone :)

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc [hunk 1]
```diff
@@ -36,65 +36,3 @@ You will not be able to fix issues that arise during the migration.
...
-----
-/usr/lib/susemanager/bin/pg-migrate-10-to-12.sh
-----
-+
-For more information about PostgreSQL database migration, see xref:db-migration-12.adoc[].
-
-
-Finally reboot the system.
-This is not part of the script.
-Initiate the reboot manually:
```
Comment (juliogonzalez) [inline modules/upgrade/pages/server-major-upgrade-uyuni.adoc:89]: This should stay, right?

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc [hunk 2]
```diff
@@ -36,65 +36,3 @@ You will not be able to fix issues that arise during the migration.
...
-
-----
-shutdown -r now
-----
-
-[NOTE]
```
Comment (juliogonzalez) [inline modules/upgrade/pages/server-major-upgrade-uyuni.adoc:95]: And this :-)

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc [hunk 3]
```diff
@@ -36,65 +36,3 @@ You will not be able to fix issues that arise during the migration.
...
-zypper ref
-----
-
-. Perform the distribution upgrade:
-+
-----
-zypper -n dup --allow-vendor-change
-----
-
-. To migrate the database to PostgreSQL 12 run the migrate script:
```
Comment (juliogonzalez) [inline modules/upgrade/pages/server-major-upgrade-uyuni.adoc:78]: And this needs to stay as well, this is not part of the script :-)

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc [hunk 4]
```diff
@@ -36,65 +36,3 @@ You will not be able to fix issues that arise during the migration.
...
-+
-----
-/usr/lib/susemanager/bin/pg-migrate-10-to-12.sh
-----
-+
-For more information about PostgreSQL database migration, see xref:db-migration-12.adoc[].
-
-
-Finally reboot the system.
-This is not part of the script.
```
Comment (juliogonzalez) [inline modules/upgrade/pages/server-major-upgrade-uyuni.adoc:88]: But this line should go.

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc (hunk 2 above)
Comment (keichwa) [reply modules/upgrade/pages/server-major-upgrade-uyuni.adoc:95]: yes, it's in again

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc (hunk 3 above)
Comment (keichwa) [reply modules/upgrade/pages/server-major-upgrade-uyuni.adoc:78]: ok, thanks for spotting!

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc (hunk 4 above)
Comment (keichwa) [reply modules/upgrade/pages/server-major-upgrade-uyuni.adoc:88]: and that one, it is in again.

File: modules/upgrade/pages/server-intro-uyuni.adoc [hunk 5]
```diff
@@ -0,0 +1,12 @@
+[[server-upgrade-intro]]
+= Upgrade the Server
+
+{productname} uses a rolling release versioning schema.
+Every Release notes provide information whether the xref:upgrade:server-minor-upgrade-uyuni.adoc[] or xref:upgrade:server-major-upgrade-uyuni.adoc[] upgrade strategy is suitable to upgrade to the next version.
```
Comment (Loquacity) [inline modules/upgrade/pages/server-intro-uyuni.adoc:5]: I don't understand this sentence.

Comment (keichwa) [reply modules/upgrade/pages/server-intro-uyuni.adoc:5]:
```suggestion
In the release notes you will find information which upgrade strategy is suitable to upgrade to the next version:
```

File: modules/upgrade/pages/server-intro-uyuni.adoc [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+Minor Upgrades::
+See xref:upgrade:server-minor-upgrade-uyuni.adoc[].
```
Comment (keichwa) [inline modules/upgrade/pages/server-intro-uyuni.adoc:8]:
```suggestion
You can consider minor upgrades as regular upgrades.
For more information, see xref:upgrade:server-minor-upgrade-uyuni.adoc[].
```

File: modules/upgrade/pages/server-intro-uyuni.adoc [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+Major Upgrades::
+Components such as the base operating system, Salt, or the PostgreSQL database will be upgraded.
```
Comment (keichwa) [inline modules/upgrade/pages/server-intro-uyuni.adoc:11]:
```suggestion
You can consider major upgrades as special upgrades.
In this case components such as the base operating system, Salt, or the PostgreSQL database will be upgraded.
```

File: modules/upgrade/pages/server-intro-uyuni.adoc [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+See xref:upgrade:server-major-upgrade-uyuni.adoc[].
```
Comment (keichwa) [inline modules/upgrade/pages/server-intro-uyuni.adoc:12]:
```suggestion
For more information, see xref:upgrade:server-major-upgrade-uyuni.adoc[].
```

File: modules/upgrade/pages/server-intro-uyuni.adoc (hunk 5 above)
Comment (keichwa) [reply modules/upgrade/pages/server-intro-uyuni.adoc]: > I don't understand this sentence.

Can you please check again now?

**Comment by keichwa** [comment]
> I'm a bit missing the context here, seems initially the actual migration script was part of the document. So yes, looks good to me :)

Thank you, Hubert!  I hope it is now consistent again.  @juliogonzalez I'm going to build a PDF now.

**Comment by keichwa** [comment]
Here's the PDF:
[uyuni_upgrade_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/4953267/uyuni_upgrade_guide.pdf)

**Comment by juliogonzalez** [comment]
> Starting with Uyuni 4.1, spacewalk-schema-upgrade is not needed
> anymore. The schema upgrade is run automatically when the spacewalk service
> is started with spacewalk-service start.

This is not correct. It  should say "Starting with Uyuni 2020.04"

**Comment by juliogonzalez** [comment]
Not for this PR, but the proxy section is completely wrong. We need it fix it as well worst case tomorrow before the end of the day so we don't block the Uyuni release even more time.

**Comment by juliogonzalez** [comment]
Finally, not you don't need to show it here, but please build the HTML as well and check that the format and the *menus* are OK.

File: modules/upgrade/pages/server-intro-uyuni.adoc [hunk 9]
```diff
@@ -0,0 +1,14 @@
+[[server-upgrade-intro]]
+= Upgrade the Server
+
+{productname} uses a rolling release versioning schema.
+In the release notes you will find information which upgrade strategy is suitable to upgrade to the next version:
```
Comment (Loquacity) [inline modules/upgrade/pages/server-intro-uyuni.adoc:5]:
```suggestion
Check the release notes for information about which upgrade strategy to use to upgrade to the next version:
```

**Comment by keichwa** [comment]
> This is not correct. It should say "Starting with Uyuni 2020.04"

ok, this is in now.  we will remove that version info sonn next time;)

**Comment by keichwa** [comment]
> Finally, not you don't need to show it here, but please build the HTML as well and check that the format and the _menus_ are OK.

HTML looks good, IMO.

**Pull request merged**